libs/orders/src/lib/components/order-list/order-list.tsx

```diff
@@ -259,18 +323,35 @@ export const OrderListTable = forwardRef<AgGridReact, OrderListTableProps>(
           field="updatedAt"
           valueFormatter={({
             value,
+            node,
           }: VegaValueFormatterParams<Order, 'updatedAt'>) => {
+            if (node?.rowPinned) {
+              return '';
+            }
             return value ? getDateTimeFormat().format(new Date(value)) : '-';
           }}
         />
         <AgGridColumn
           colId="amend"
           headerName=""
           field="status"
-          cellRenderer={({ data }: VegaICellRendererParams<Order>) => {
+          cellRenderer={({ data, node }: VegaICellRendererParams<Order>) => {
+            if (node?.rowPinned) {
```

Comment:
chore (non blocking): the bottom pinned row has a background hover color, can we remove it?

Also having tested it out, I don't think its clear enough that this will close_all_ orders and not just orders for the given market. I think we should make the orders on the market page be only for that market. The orders tab on the portfolio page can continue to show all orders across all markets.

Reply:
We are showing all orders on market page not filtered by market Id
We can filter them out and show only orders for the market when filters will be available on API.